src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -34,13 +34,13 @@ namespace Microsoft.IdentityModel.Tokens
     /// Creates <see cref="SignatureProvider"/>s by specifying a <see cref="SecurityKey"/> and algorithm.
     /// <para>Supports both <see cref="AsymmetricSecurityKey"/> and <see cref="SymmetricSecurityKey"/>.</para>
     /// </summary>
-    public class SignatureProviderFactory
+    public class CryptoProviderFactory
```

Comment:
Honestly, if you think the new name is a better name, I won't fight to keep the old one, as it's just a cosmetic change :smile: